modules/tests/src/test/scala/MonixParallelTests.scala

```diff
@@ -18,16 +18,14 @@ package freestyle.tests
 
 import freestyle.implicits._
 import monix.eval.Task
-import monix.cats._
-import monix.eval.Task.nondeterminism
 import monix.execution.Scheduler.Implicits.global
 import org.scalatest.{Matchers, WordSpec}
 import scala.concurrent.Await
 import scala.concurrent.duration.Duration
 
 class MonixParallelTests extends WordSpec with Matchers {
   "Applicative Parallel Support" should {
-    "allow non deterministic execution when interpreting to monix.eval.Task" in {
+    "allow non deterministic execution when interpreting to monix.eval.Task" ignore {
```

Comment:
If this tests are not longer valid we should remove them.

Reply:
The module is not aggregated in the `root` module, should I remove it?